processing/src/main/java/org/apache/carbondata/processing/store/TablePage.java

```diff
@@ -182,7 +183,11 @@
       } else {
         page = ColumnPage.newPage(columnPageEncoderMeta, pageSize);
       }
-      page.setStatsCollector(PrimitivePageStatsCollector.newInstance(dataTypes[i]));
+      if (dataTypes[i] != DataTypes.BINARY) {
+        page.setStatsCollector(PrimitivePageStatsCollector.newInstance(dataTypes[i]));
+      } else {
+        page.setStatsCollector(new DummyStatsCollector());
+      }
       measurePages[i] = page;
     }
 
```

Comment:
better to update the document, such as "Following Data Types are not Supported for Local Dictionary"